tests/org.eclipse.emf.mwe2.language.tests/META-INF/MANIFEST.MF

```diff
@@ -21,7 +21,8 @@ Require-Bundle: org.eclipse.emf.mwe2.language,
  org.eclipse.ui.workbench;resolution:=optional,
  org.eclipse.xtext.xbase.lib,
  org.eclipse.xtext.testing,
- org.eclipse.xtext.ui.testing
+ org.eclipse.xtext.ui.testing,
+ org.eclipse.xtext.generator
```

Comment:
i wonder if we can test this without using the old generator we want to get rid of.
afaik the new generator has a few deprecated stuff too e.g.
in XtextGeneratorLanguage or wizard